codebase2/codebase-sqlite/U/Codebase/Sqlite/NamedRef.hs

```diff
@@ -49,11 +50,6 @@ instance (FromRow ref) => FromRow (NamedRef ref) where
     ref <- fromRow
     pure (NamedRef {reversedSegments, ref})
 
-toRowWithNamespace :: (ToRow ref) => NamedRef ref -> [SQLData]
```

Comment:
This is just unused now (it got folded into the ToRow instance when the migration was merged)